exporter.py

```diff
@@ -109,4 +124,5 @@ def initialize_collector_by_name(class_name, logger):
     logger = logging.getLogger('vrops-exporter')
     options = parse_params(logger)
     collectors = list(map(lambda c: initialize_collector_by_name(c, logger), options.collectors))
+    collector_globals = collectors
```

Comment:
You can simply make the `collector` variable global without creating a new variable. Then it doesn't have to be passed through everywhere.
```suggestion
    global collectors
    collectors = list(map(lambda c: initialize_collector_by_name(c, logger), options.collectors))
```

Reply:
Good catch, since it involves the change of all previous occurrences I'm going to put this in the next commit.